File: solutions/beecrowd/1041/1041.c
#include <stdio.h>

int main() {
    float x, y;

    while (scanf("%f %f", &x, &y) != EOF) {
        if (x == 0.0 && y == 0.0) {
            printf("Origem\n");
        } else if (x == 0.0 && y != 0.0) {
            printf("Eixo Y\n");
        } else if (y == 0.0 && x != 0.0) {
            printf("Eixo X\n");
        } else if (x > 0.0) {
            if (y > 0.0) {
                printf("Q1\n");
            } else {
                printf("Q4\n");
            }
        } else {
            if (y > 0.0) {
                printf("Q2\n");
            } else {
                printf("Q3\n");
            }
        }
    }

    return 0;
}

File: solutions/beecrowd/1041/1041.py
import sys

for line in sys.stdin:
    x, y = map(float, line.split())
    if not x and not y:
        print('Origem')
    elif not x and y:
        print('Eixo Y')
    elif x and not y:
        print('Eixo X')
    elif x > 0.0:
        if y > 0.0:
            print('Q1')
        else:
            print('Q4')
    else:
        if y > 0.0:
            print('Q2')
        else:
            print('Q3')
